script/hassfest/quality_scale_validation/parallel_updates.py

```diff
@@ -0,0 +1,36 @@
+"""Enforce that the integration set Parallel updates.
+
+https://developers.home-assistant.io/docs/core/integration-quality-scale/rules/parallel-updates
+"""
+
+import ast
+
+from homeassistant.const import Platform
+from script.hassfest.model import Integration
+
+
+def _has_parallel_updates_defined(module: ast.Module) -> bool:
+    """Test if the module defines `PARALLEL_UPDATES` constant."""
+    return any(
+        type(item) is ast.Assign and item.targets[0].id == "PARALLEL_UPDATES"
+        for item in module.body
+    )
+
+
+def validate(integration: Integration) -> list[str] | None:
+    """Validate that the integration has a reauthentication flow."""
```

Comment:
Reference to reauthentication should be changed to parallel_updates
`"""Validate that the integration has a PARALLEL_UPDATES set."""`